### PR TITLE
update/SOF 7469

### PR DESCRIPTION
**Pull request by VsevolodX**
- update: create one material based on parameters
- update: add create defects

### CODE REVIEW

File: other/materials_designer/create_point_defect.ipynb [hunk 1]
```diff
@@ -10,17 +10,10 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```
Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:10]: <p>List numbering is broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:10]: <pre class="ql-syntax" spellcheck="false">Line #9.    DEFECT_CONFIGS = [
</pre><p>Put a link to the defect configuration explanation in made (perhaps not in comment, but add a sentence below 1.1)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:10]: <pre class="ql-syntax" spellcheck="false">Line #5.    for defect in DEFECT_CONFIGS:
</pre><p>for config in DEFECT_CONFIGS</p><p> </p><p>The if/else statement should go to PointDefectConfiguration constructor overloaded <code>from_dict</code> or something like that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (VsevolodX) [reply other/materials_designer/create_point_defect.ipynb:10]: Only on review, Not in the JL notebooks themselves

Comment (timurbazhirov) [reply other/materials_designer/create_point_defect.ipynb:10]: <p>Not addressed</p>

Comment (timurbazhirov) [reply other/materials_designer/create_point_defect.ipynb:10]: <p>We should either update the class docstring to have approximate coordinate and site_id or put links to specific </p>

File: other/materials_designer/create_point_defect.ipynb [hunk 2]
```diff
@@ -9,18 +9,11 @@
     "\n",
```
Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #22.    ]  # List of dictionaries with defect parameters
</pre><p>Move to the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:9]: <p>Initialize Configuration and Builder parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (timurbazhirov) [inline other/materials_designer/create_point_defect.ipynb:9]: <p>Visualize Result(s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

File: other/materials_designer/specific_examples/defect_creation_point_substitution_graphene.ipynb [hunk 3]
```diff
@@ -0,0 +1,292 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/specific_examples/defect_creation_point_substitution_graphene.ipynb:1]: <p>1.3. Load input material</p><p> </p><p>Explain that the material comes from a set of test fixtures</p><p> </p><p>Create a task to update Standata and use it here instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: other/materials_designer/create_point_defect.ipynb (hunk 1 above)
Comment (VsevolodX) [reply other/materials_designer/create_point_defect.ipynb:10]: Updated the docstring and point link to that line

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**